sysboundary/ionosphere.cpp

```diff
@@ -3768,6 +3770,18 @@ namespace SBC {
          Real temperature = 0;
          Real density = 0;
          switch(boundaryVDFmode) {
+            case ForceL2EXB:
+               {
+               // EXB forcing is assigned to the L2 Neighbour cells here, so they can update their VDFs in acceleration
+               const vector<CellID>& closestCells = getAllClosestNonsysboundaryCells(cellID);
+               for (CellID celli : closestCells) {
+                  mpiGrid[celli]->parameters[CellParams::BULKV_FORCING_X] += vDrift[0];
+                  mpiGrid[celli]->parameters[CellParams::BULKV_FORCING_Y] += vDrift[1];
+                  mpiGrid[celli]->parameters[CellParams::BULKV_FORCING_Z] += vDrift[2];
+                  mpiGrid[celli]->parameters[CellParams::FORCING_CELL_NUM]+=1;
+               }
+               // Fall through, to handle L1 in the same way as fixed moments
```

Comment:
This is never called for NOT_SYSBOUNDARY L2 cells, we need to add that call somewhere.